src/app/api/run/[runId]/upload/route.test.ts

```diff
@@ -59,4 +50,19 @@ describe('POST /api/run/[runId]/upload', () => {
         const response = await POST(req, { params })
         expect(response.status).toBe(400)
     })
+
+    it('should return an error if no runID is provided', async () => {
+        const formData = new FormData()
+
+        // Mock the NextRequest
+        const req = {
+            formData: async () => formData,
+        } as NextRequest
+        const params = {}
+        // @ts-ignore
+        const response = await POST(req, { params })
+        expect(response.status).toBe(400)
+        const data = await response.json()
+        expect(data.error).toBe('Missing runId')
```

Comment:
I dunno, a 400 might be enough since I don't think anyone else depends on the exact string that's returned